spec/models/vote_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.describe Vote, type: :model do
+  it {should belong_to(:review)}
+
+  it {should belong_to(:user)}
+
+  it {should have_valid(:value).when(1, 0, -1)}
+  it {should_not have_valid(:value).when(nil, '', 2, -2, 'a')}
+end
```

Comment:
I understand you're setting this up for when you add the tests, but then this should be left out of the PR until the tests are complete. We don't want unfinished code being merged to Master.

Reply:
This test is for a join table which has three columns. Two for associations and one for storing a value (a vote for each unique relationship between a user and a review). We intended for this test to be complete, but let us know what else we need for this to be a complete test.